modules/client-configuration/pages/client-operations.adoc

```diff
@@ -1,10 +1,8 @@
 [[client.operations]]
 = Client Operations
 
-In addition to registering, upgrading, or deleting clients other operations can be performed.
+In addition to registering, upgrading, installing software, or deleting clients other operations can be performed.
+{productname} clients can be managed individually, or organized in groups using System Set Manager, System Groups, or Configuration Management.
```

Comment:
For me it's mot really clear what does it mean to be `organized in groups using ..., or Configuration Management`? Sounds like Configuration Management can organize systems to the groups. Sounds very confusing. I see the phrase was before the change, but for me it sounds like we need to fix it.

Reply:
yes, thanks for feedback.  Please check again, I tried to simplify and enhance it.  This text is meant as an intro to the following sections that will actually explain the features listed here.  If sth is missing or still unclear do not hesitate to add proposals or questions.

Reply:
Thank you. Now it sounds correct and not causing any confusions. :+1: